testcases/MachineConfig.py

```diff
@@ -59,10 +59,33 @@ def setUp(self):
         else:
             self.skipTest("Functionality is supported only on LPAR")
 
+    def validate_secureboot_parameter(self, machine_config):
```

Comment:
Had renamed & moved this function from MachineConfig.py to OpTestUtil.py based on inputs around - https://github.com/open-power/op-test/pull/778#issuecomment-1698506238 

Nonetheless, if we all are good with having it back here - not a stopper - good to go!